modules/installation-osp-enabling-swift.adoc

```diff
@@ -17,6 +17,13 @@ If link:https://access.redhat.com/documentation/en-us/red_hat_openstack_platform
 If Swift is present and you want to use it, you must enable access to it. If it is not present, or if you do not want to use it, skip this section.
 ====
 
+[IMPORTANT]
+====
+Early versions of {rh-openstack} 17.0 set the `rgw_max_attr_size` parameter of Ceph RGW to 256 characters. This setting causes issues with uploading container images to the {product-title} registry. You must set the value of `rgw_max_attr_size` to at least 1024 characters.
+
+Before installation, check if your {rh-openstack} deployment is affected by this problem. If it is, reconfigure Ceph RGW.
```

Comment:
```suggestion
Before installation, check if your {rh-openstack} deployment is affected by this problem. If it is, reconfigure Ceph RGW.
```
Is there documentation that succinctly describes how to check for this issue?

Reply:
I don't have any and IIRC Ceph allows you to set options in various places, so this might be too complicated to explain here.